output.txt

```diff
@@ -1,6 +1,27 @@
 You are now connected to database "nc_games" as user "benjismith1990".
- review_id |    title     | category  |   designer   |  owner   |                                                                                                                                                                                                                     review_body                                                                                                                                                                                                                     |                                 review_img_url                                 |       created_at        | votes 
------------+--------------+-----------+--------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+-------------------------+-------
-         2 | JengARRGGGH! | dexterity | Leslie Scott | grumpy19 | Few games are equiped to fill a player with such a defined sense of mild-peril, but a friendly game of Jenga will turn the mustn't-make-it-fall anxiety all the way up to 11! Fiddly fun for all the family, this game needs little explaination. Whether you're a player who chooses to play it safe, or one who lives life on the edge, eventually the removal of blocks will destabilise the tower and all your Jenga dreams come tumbling down. | https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700 | 2021-01-18 10:01:41.251 |     5
-(1 row)
+    owner     |                        title                         | review_id |    category     |                                        review_img_url                                        |       created_at        | votes |        designer        | count 
```

Comment:
I'd git ignore this, but as it has now been committed you'd need to remove it from github entirely to stop it being tracked.

Reply:
that should already be in the ignore file? strange!